### PR TITLE
Desmume: init ao 0.9.11

### PR DESCRIPTION
**Pull request by AndersonTorres**

### CODE REVIEW

**Comment by AndersonTorres** [comment]
Can someone explain why Travis is with an internal error?

File: pkgs/misc/emulators/desmume/default.nix [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{ stdenv, fetchurl
+, pkgconfig, libtool, intltool
+, libXmu
+, lua
+, agg, alsaLib, soundtouch, openal
+, desktop_file_utils
+, gtk2, gtkglext, libglade, pangox_compat
+, mesa, mesa_glu, libpcap, SDL, zziplib }:
+
+with stdenv.lib;
```
Comment (pSub) [inline pkgs/misc/emulators/desmume/default.nix]: As far as I can see you use stuff from stdenv.lib only in meta, thus it makes sense to put those into scope only there.

**Comment by pSub** [comment]
I've restarted the travis build. Lets see what happens.

**Comment by pSub** [comment]
The build succeeds now.

File: pkgs/misc/emulators/desmume/default.nix (hunk 1 above)
Comment (AndersonTorres) [inline pkgs/misc/emulators/desmume/default.nix]: It is my particular idiom :)

In general, I noted when I build my expressions I always needed "stdenv.lib" in the most outside scope. So, that's it.

**Pull request merged**